src/lib/layer-manager.js

```diff
@@ -173,6 +170,26 @@ export default class LayerManager {
     });
   }
 
+  pickLayerByBoundingBox({x, y, width, height}) {
```

Comment:
No. `pickLayer()` is used by mouse event picking (returns a single info object) and `pickLayerByBoundingBox()` is used by the query API (returns an array of info objects).